src/iop/temperature.c

```diff
@@ -81,8 +82,33 @@ typedef struct dt_iop_temperature_global_data_t
 {
   int kernel_whitebalance_4f;
   int kernel_whitebalance_1f;
+  double RGB_to_CAM[4][3], CAM_to_RGB[3][4];
```

Comment:
I'm not sure we can/should use global data for that, it it global, so there are likely to be some problems..
I would recommend storing it in dt_iop_temperature_data_t in commit_params()